### DISCUSSION TITLE
tmp folder is read only

### DISCUSSION THREAD

**Discussion by Casraw** [cloudnative-pg/cloudnative-pg#704] [General]
Hello together,

we want to use the cloudnative-pg. Now we found out that the tmp folder is read only. Is there any possibility to make it rw and why is this like that?

We want to pipe a file trough kubectl into the postgres cluster. And this is failing.

Thanks, Casraw

**Comment by gbartolini** [comment]
Hi Casraw,

do you mind explaining the use case? What is your goal?

Our approach is for the operand container to be read-only, for security reasons.

Thanks,
Gabriele

**Reply by Casraw** [reply]
The goal is that we can pipe something into psql. Like:

```
(self._base_cmd()
 .arg('bash', '-c')
 .arg(
    f"PGUSER={self.owner} PGDATABASE={self.database} PGPASSWORD={self.admin_password} "
    f"psql --quiet -o {remote_output_file} {psql_args} <<'EOF'\n"
    f"{cmd}\n"
    "EOF\n"
)
```

and this makes problems. Also with kubectl directly it is not possible to pipe something in it.

**Reply by Casraw** [reply]
We now moved our system to not use here files anymore. Therefore it is obsolet.

**Comment by sxd** [comment]
@Casraw I'm turning this into a discussion

**Reply by Casraw** [reply]
Thanks :-)
